### PR TITLE
Add social links + youtube embed component

### PR DESCRIPTION
**Pull request by Bianca-Morris**
**Changelog:**
- Add new "Video Tutorials" external link to the help menu (goes to the YouTube channel)
- Add GitHub and YouTube social icon links to the footer
- Create a new component for embedding YouTube videos
   - Lazy loads the YouTube video by showing the poster first, then pre-connecting to YouTube on hover, and loading the full iframe with the video on click (needed for performance, since we will sometimes be loading many videos on one static page)
   - Allows auto-play (in which case lazy load doesn't happen) or not
   - Note: does NOT support playlists currently (would need more updates for that)
- Adjust content security policy (Will already approved these edits) to allow pulling videos and thumbnail images from YouTube directly
- Add `YoutubeVideoEmbed` component as a "placeholder" for use in JSX static sections
- (Victoria's PR https://github.com/dbmi-bgm/cgap-portal/pull/675) - Update a bunch of static sections to use JSX
- (Victoria's PR https://github.com/dbmi-bgm/cgap-portal/pull/675) - Update master inserts to use new JSX files

**Deployment Notes:**
Deployed currently to `cgap-devtest`. Header and Footer updates are visible on every page. There's a link to an example video embed on here under "Filling the Spreadsheet": https://cgap-devtest.hms.harvard.edu/help/submission/accessioning. 

**Screenshots:**
[Image from Gyazo](https://gyazo.com/862afe685bba43e38fd8f30ea58c6310)
[Image from Gyazo](https://gyazo.com/545c389dae456abd010dc2e71262cf29)
This was generated using this code in a JSX static section:
```
{/** Note: YoutubeVideoEmbed will take up full width, and default to 16:9 aspect ratio;
might make a wrapper component for this in the future to make it easier to set a max width. **/}

<h2>Genotype First Analysis</h2>
<div style={{ maxWidth: "500px"}}> 
  <YoutubeVideoEmbed
    shouldAutoplay={false}
    videoID="HdOHFM7BjWk"
    videoTitle="Genotype First Analysis"
  />
</div>

<h2>Structural Variant Analysis</h2>
<div style={{ maxWidth: "500px"}}>
  <YoutubeVideoEmbed
    shouldAutoplay={false}
    videoID="hhD9yTfTWVQ"
    videoTitle="Structural Variant (SV) Analysis"
  />
</div>
```

### CODE REVIEW

**Review by shannonehmsen — APPROVED** [review]
Links in the header and footer look great and work as expected. 

I was only able to see one video inside of the help sections for case submission. I am assuming because this is devtest they aren't all up? Please let me know if I should look elsewhere or if you need my input on the correct location for all remaining videos. 

The functionality of the video and size was great, there was no waiting time for the video to load or appear on screen. I also liked that the video was muted by default.

**Review by utku-ozturk — APPROVED** [review]
looks good to me!

**Review by dmichaels-harvard — APPROVED** [review]
Looks good to me.

**Review by willronchetti — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 # Note: Various modules refer to this system as "encoded", not "cgap-portal".
 name = "encoded"
-version = "12.0.0"
+version = "12.1.0"
```
Comment (vstevensf) [inline pyproject.toml:4]: Just double check before merging but I'm sure you're right :)

**Review by vstevensf — APPROVED** [review]
Looks great. Thanks for all your help on this!

**Pull request merged**